sbt-plugin/src/main/scala/org/scalajs/sbtplugin/LinkerImpl.scala

```diff
@@ -75,6 +75,18 @@ object LinkerImpl {
     private val parentPrefixes = List(
       "java.",
       "scala.",
+      /**
+       * This is a workaround for java.lang.NoClassDefFoundError: jdk/internal/reflect/MethodAccessorImpl,
+       * which happened when compiling multiple Scala.js projects:
+       * https://github.com/scala-js/scala-js/issues/3921
+       *
+       * The cause is java.lang.reflect.Method becomes invalid after 14 "invokes".
+       * The default threshold of -Dsun.reflect.inflationThreshold is 15:
+       * https://www.ibm.com/developerworks/community/forums/html/threadTopic?id=77777777-0000-0000-0000-000013894489&ps=25
+       *
+       * So we will delegate the access to sun.* packages to the parent class loader instead of using reflection.
+       */
```

Comment:
Please:

- Put the workaround at the end of the list.
- Adjust the comment as pointed out in https://github.com/scala-js/scala-js/issues/3921#issuecomment-575553465
- Consider only adding the prefix "sun.reflect."
- Also add "jdk.internal.reflect." (judging from the stack trace in #3921) or verify this workaround works for the airframe CI.

Reply:
So on second reading, I think this comment still needs improvement. Notably, the second section doesn't "connect up" to the first one.

Things that at the moment are unclear from the comment IMHO:
1. Why is `"sun.reflect."` there as well?
2. If `java.lang.reflect.Method` becomes invalid, why does this fix it? Why do we not need to recreate a new `jlr.Method`?
3. Its non obvious that the prefixes are implementation details of the relevant JDKs (this is related to the first point). So if somebody sees a `NoClassDefFoundError: moon.reflect.MethodAccessorImpls` missing, this is probably the same issue.

I think point 1 and 3 can be addressed by making it clear that this is an implementation detail and `jdk.internal.reflect.MethodAccessorImpl` and `sun.reflect.MethodAccessorImpl` are the *currently* known affected classes.

Point 2 is a bit more tricky: I think we need to either explain the thing in full, or omit explanation of *why* this happens and simply describe

- trigger (call `invoke` `sun.reflect.inflationThreshold` times on same `Method` instance)
- symptom (`NoClassDefFoundError`)
- mitigation (add prefix here)

and refer to the bug, letting the reader reconstruct the details by themselves.